src/partitionBy.js

```diff
@@ -0,0 +1,2 @@
+var _partitionBy = require('./internal/_partitionBy');
+module.exports = _partitionBy;
```

Comment:
It is not an :alien: Sorry my comments are all over the place, the only note I made was in `src/internal/_partitionBy.js` that I was following this contract: https://clojuredocs.org/clojure.core/partition-by. It certainly seems to me that how `partition` is defined in Ramda can be useful, just so happened I needed the array to be partitioned exactly as it is defined by clojure's partition-by. I think if Ramda is going to include both, one of the names will have to change as the names `partition` and `partitionBy` do not give much insight into the actual difference.